client/packages/host/src/components/Footer/Footer.tsx

```diff
@@ -77,10 +78,14 @@ export const Footer: React.FC = () => {
       {user ? (
         <>
           <Divider />
-          <PaddedCell>
-            <UserIcon sx={iconStyles} />
-            <Typography sx={textStyles}>{user.name}</Typography>
-          </PaddedCell>
+          <UserDetails>
+            <PaddedCell>
+              <UserIcon sx={iconStyles} />
+              <Tooltip title={`${user.firstName} ${user.lastName}`}>
```

Comment:
can you please use `getLocalisedFullName` to account for empty strings etc

Reply:
Updated here: dbcdd53d553899815728e0849f76c11071c0009d